src/main/java/uk/gov/register/resources/PreviewEntriesResource.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.register.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import io.dropwizard.views.View;
+import uk.gov.register.core.Entry;
+import uk.gov.register.core.RegisterReadOnly;
+import uk.gov.register.views.ViewFactory;
+import uk.gov.register.views.representations.ExtraMediaType;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.Collection;
+import java.util.Collections;
+
+@Path("/")
+public class PreviewEntriesResource {
+
+    private static final int START = 0;
+
+    protected final ViewFactory viewFactory;
+    private final RegisterReadOnly register;
+
+    @Inject
+    public PreviewEntriesResource(final ViewFactory viewFactory, final RegisterReadOnly register) {
```

Comment:
This is probably more of an individual style point, but do we need to specify `final` here when we assign these variables to `final` class variables?

Reply:
This was already discussed. Final stay

Reply:
Discussed.